tools/dashboard/package.json

```diff
@@ -16,10 +16,13 @@
     "@angular/core": "^4.2.2",
     "@angular/forms": "^4.2.2",
     "@angular/http": "^4.2.2",
+    "@angular/material": "^2.0.0-beta.6",
```

Comment:
If we make this point to the build snapshot it could help give us a heads-up if something goes wrong